packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -125,9 +124,10 @@ class MenuItem extends React.Component {
     return (
       <li
         className={cx('list-item')}
+        role="none"
```

Comment:
i'm curious, what happens if we don't explicitly set role to none here?

Reply:
If role is not set explicitly it gives an axe issue `Ensures <li> elements are used semantically`. Also here is [link](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html) for reference